packages/@tailwindcss-postcss/src/__snapshots__/index.test.ts.snap

```diff
@@ -37,9 +37,9 @@ exports[`\`@import 'tailwindcss'\` is replaced with the generated CSS 1`] = `
     -webkit-text-size-adjust: 100%;
     tab-size: 4;
     line-height: 1.5;
-    font-family: var(--default-font-family, ui-sans-serif, system-ui, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol", "Noto Color Emoji");
-    font-feature-settings: var(--default-font-feature-settings, normal);
-    font-variation-settings: var(--default-font-variation-settings, normal);
+    font-family: var(--default-font-family);
+    font-feature-settings: var(--default-font-feature-settings);
+    font-variation-settings: var(--default-font-variation-settings);
```

Comment:
This might be controversial but the way the `--theme(…)` function works is that it omits the fallback when the theme is found within the design system (this is necessary to work with `@reference` where we want to append a fallback to the theme `var(…)` declaration based on the configured theme value.

As a result, this will now omit the the fallback values here. But if you scroll down a bit, `--font-sans` and `--default-font-family` CSS variables will be emitted anyways so we _know_ the fallback is not necessary thus save some bytes in the resulting CSS file.